README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# **Samuel Northrup - HW01 - MACS 30500**
+
+## *Biographical Information*
+I grew up in rural Kentucky, and became interested in the Middle East after studying Arabic in Amman, Jordan. I loved seeing and learning about different cultures and worldviews. I spent the rest of my time in high school and college learning Arabic so that one day I could return to the region and explore the culture even more. 
+
+## *Education and Work*
```

Comment:
Bolding and heading is a very effective way to add extra emphasis on top of emphasis. If that's what you're going for, great job here. It might be overwhelming to use this style in every instance of a header, maybe.